assets/src/components/v2/pre_fare_single_screen_alert.tsx

```diff
@@ -360,6 +369,16 @@ const PreFareSingleScreenAlert: React.ComponentType<
         />
       );
       break;
+    case !disruption_diagram:
```

Comment:
Might be unnecessarily nitpicky, but can this be `disruption_diagram === undefined` instead so all the cases consistently check for equality? This gets the point across though so totally up to you.